### PR TITLE
Storage: Port test changes from cr/250550507

### PR DESCRIPTION
**Pull request by schmidt-sebastian**
Manual port since we don't have Copybara set up yet.

https://critique.corp.google.com/#review/250550507

### CODE REVIEW

**Comment by schmidt-sebastian** [comment]
Unassigning since there is more cleanup to port (the CL was updated).

**Comment by schmidt-sebastian** [comment]
> Unassigning since there is more cleanup to port (the CL was updated).

The new changes in the CL were already applied to this repo.

File: firebase-storage/src/test/java/com/google/firebase/storage/PathingTest.java [hunk 1]
```diff
@@ -164,7 +164,7 @@ public void dotdotTest() throws Exception {
         FirebaseStorage.getInstance()
             .getReferenceFromUrl("gs://benwu-test1" + ".storage.firebase.com/child/../");
 
-    assertEquals(ref.getName(), "..");
+    assertEquals("..", ref.getName());
```
Comment (rsgowman) [inline firebase-storage/src/test/java/com/google/firebase/storage/PathingTest.java]: I have to look up this ordering, every. single. time. (FTR, you've done the correct thing.)

However, if you're going to go to the effort of fixing this, you might consider migrating to assertThat instead. It makes it much more obvious.

Optional.

Comment (schmidt-sebastian) [reply firebase-storage/src/test/java/com/google/firebase/storage/PathingTest.java]: Keeping it as is to match the base CL.

**Pull request merged**